stack.py

```diff
@@ -112,6 +114,18 @@ def eval_program(program):
             if dup_depth > len(stack):
                 raise IndexError
             stack.extend(stack[-dup_depth:])
+        elif instr.op == 'jump':
+            if instr.args:
+                jump_distance = instr.args[0]
+            else:
+                jump_distance = stack[-1]
+                if 'quiet' not in instr.prefix:
+                    stack.pop()
+            # We jump 1 less than the argument since we already incremented it
+            # at the beginning of the loop.
+            current_instr += int(jump_distance) - 1
```

Comment:
If the jump distance is so large to be negative, then the next instruction to be read from the end of the program. It will also cause an infinite loop since the while loop condition will never be false.